docs/source/install.md

```diff
@@ -24,6 +24,14 @@ This requires compatible versions of `GDAL` and `numpy` from `conda-forge` for
 raw I/O support and `geopandas` and their dependencies for GeoDataFrame
 I/O support.
 
+By default, the `GDAL` package on conda-forge currently includes LIBKML and
+Spatialite. However, you may need to install additional drivers by installing
+the associated [conda-forge package](https://gdal.org/en/latest/download.html#conda):
+
+-   `libgdal-arrow-parquet`
+-   `libgdal-pg`
+-   `libgdal-xls`
```

Comment:
```suggestion
-   `libgdal-arrow-parquet` ((Geo)Parquet and (Geo)Arrow IPC)
-   `libgdal-pg` (PostgreSQL / PostGIS)
-   `libgdal-xls` (XLS - MS Excel format)
```

(for `pg` I think it is certainly valuable to add the full name for clarity, the others might be clear already)